.ci/e2e.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#####
+# This script is created to install kuberhealthy with a few basic checks in a minikube cluster.
+# In the long run I hope that we can use it to run test cases.
+#####
+
+# Set NS
+NS=kuberhealthy
+name=kuberhealthy
+
+# Create namespace
+kubectl create namespace $NS
+
+# Sometimes the kuberhealthy resources get's created...
+sleep 2
+
+# Create kuberhealthy crd etc.
+kubectl create -f .ci/kuberhealthy-e2e.yaml
```

Comment:
Hmm I'm wondering if we could use helm to install kuberhealthy instead -- this could make the e2e test more configurable cause then instead of having to add `.ci/kuberhealthy-e2e.yaml` file, its just a `values.yaml` file that can be configured to enable or disable certain external checks while running e2e testing. I know the default kuberhealthy configuration only enables those 3 checks, but it'd be nice for certain PRs that are making changes to external checks to add them to the e2e testing. Not sure how configurable github actions can be for that :woman_shrugging:

Reply:
It's a good point, I will play around with helm and provide a update to this PR.

Sadly the AMI check will always be hard to test.
I guess we could create a IAM user in some test environment where we can setup some data, but it's an edge case so let's focus on the low hanging fruit first.


We could trigger to all PR:s and do some ninja script to parse out what files have changed since the last commit and only trigger e2e tests due to those changes but it would be a pain to maintain.

I don't see a big issue with triggering al the e2e tests on all the PR:s, sure each run will take a few minutes longer but I would say it's worth it compared to maintaining strange scripts.
But to make sure that we don't start e2e tests unless we have to I think we should trigger 
go test ./... firsts so if someone sen't in a PR that wasn't possible to build/test we would just cancel the job instead of running all of the tests.

Reply:
Yeah, I think ami-test is also environment specific so I'm okay skipping that. 
I agree with e2e only being triggered after we know the PR can pass tests / build!